api/internal/handler/nft_handler.go

```diff
@@ -70,3 +80,133 @@ func NewUpdateNftBackstoryHandler(nftTokenRepository repository.NftTokenReposito
 		c.JSON(http.StatusOK, nftToken)
 	}
 }
+
+func NewGetNftsHandler(nftTokenRepository repository.NftTokenRepository) func(c *gin.Context) {
+	return func(c *gin.Context) {
+
+	}
+}
+
+func NewUpdateNftHandler(nftTokenRepository repository.NftTokenRepository, client *ethclient.Client) func(c *gin.Context) {
```

Comment:
for all API handlers, better include a link to the API doc on postman in the comments so that we can easily find the document of the API. you can do it later to add those links to all API handlers.

Reply:
Ok will add in a separate PR